### PR TITLE
Update plover to version 3.0.0

### PR DESCRIPTION
**Pull request by arxanas**
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download plover` is error-free.
- [x] `brew cask style --fix plover` left no offenses.

### CODE REVIEW

File: Casks/plover.rb [hunk 1]
```diff
@@ -1,16 +1,18 @@
 cask 'plover' do
-  version '2.5.8'
-  sha256 'a8bbeddb5d6fb8d4499844257035edd62e431a4f1eb7959bdf17a15c1cbe12de'
+  version '3.0.0'
+  sha256 '89505ebad2700e38d57b5c710550dac0ad4b9a44bd1e15ba8b1f162f0b464694'
 
   # github.com is the official download host per the vendor homepage
```
Comment (adidalal) [inline Casks/plover.rb]: Needs a `# github.com/openstenoproject/plover was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (arxanas) [inline Casks/plover.rb]: Should it be `github.com/openstenoproject` or `github.com/openstenoproject/plover`? I pushed the former with the reasoning that it is the vendor in "the smallest possible portion of the URL that uniquely identifies the app or vendor".

Comment (vitorgalvao) [inline Casks/plover.rb]: Should be `github.com/openstenoproject/plover`.

Comment (arxanas) [inline Casks/plover.rb]: Updated.

**Pull request merged**